WordPress/src/main/java/org/wordpress/android/ui/uploads/PostEvents.java

```diff
@@ -4,18 +4,18 @@
 
 public class PostEvents {
     public static class PostUploadStarted {
-        public final int mLocalBlogId;
+        public final PostModel post;
```

Comment:
We need to access more than the siteId to be able to refresh the relevant row in post list, that's why this change was implemented for both `PostUploadStarted` and `PostUploadCanceled`.